### DISCUSSION TITLE
Sometimes admin panel slow load

### DISCUSSION THREAD

**Discussion by amigolj** [Laravel-Backpack/community-forum#1227] [General] [answered]
Today I checked on 4 projects where I have laravel backpack they are all on different servers and long loading login window and then switching between sections of admin panel sometimes stop working links inside the admin panel that is clicking on them nothing happens. I noticed this 2 times during half a year

[a comment by an automated account is not shown]

**Reply by pxpm** [reply, marked as answer]
Also to add, make sure you don't have `Basset` in dev mode in production. In case you do, basset is fetching all the necessary assets everytime a page is loaded. 
You can make sure that doesn't happen by adding `BASSET_DEV_MODE=false` on your production `.env` file. 

Cheers

**Reply by amigolj** [reply]
Thanks, I'll try it
